src/ViewModel/HighlightItem.php

```diff
@@ -21,7 +21,7 @@ final class HighlightItem implements ViewModel
     private $meta;
     private $image;
 
-    public function __construct(array $subjects, Link $title, Meta $meta, Picture $image, string $summary, string $authors = null)
+    public function __construct(array $subjects, Link $title, Meta $meta, Picture $image, string $summary = null, string $authors = null)
```

Comment:
We will need a change to the pattern-library as summary is currently required for the highlight-item template.

https://github.com/elifesciences/pattern-library/blob/develop/source/_patterns/01-molecules/components/highlight-item.yaml#L24

Once we make the change to pattern-library then we need to adjust the ViewModel and the accompanying test.